src/codegen/generate.ts

```diff
@@ -444,18 +437,36 @@ export default class ApiGenerator {
     return cg.keywordType.any;
   }
 
+  /**
+   * Creates literal type (or union) from an array of values
+   */
+  getTypeFromEnum(values: any[]) {
```

Comment:
Very minor point, but if you could make this `any[]` an `unknown[]` as well, I think that would be better.
(I understand your point about `schema.enum` being an `any[]`, but I rather like the `unknown` type :grin: ).

Reply:
will add this and release 🚀